packages/web-app-files/tests/unit/components/FilesList/ListInfo.spec.js

```diff
@@ -0,0 +1,83 @@
+import ListInfo from '../../../../src/components/FilesList/ListInfo.vue'
+import GetTextPlugin from 'vue-gettext'
+import { createLocalVue, shallowMount } from '@vue/test-utils'
+import Vuex from 'vuex'
+
+const localVue = createLocalVue()
+localVue.use(Vuex)
+localVue.use(GetTextPlugin, {
+  translations: 'does-not-matter.json',
+  silent: true
+})
+describe('ListInfo', () => {
+  function getWrapper(props = {}) {
+    return shallowMount(ListInfo, {
+      localVue,
+      propsData: {
+        files: 2,
+        folders: 3,
+        ...props
+      }
+    })
+  }
+  describe('files and folders prop', () => {
+    const wrapper = getWrapper()
+    const itemElement = wrapper.find('p')
+
+    it('should set data test attributes', () => {
+      expect(itemElement.attributes()['data-test-items']).toBe('5')
+      expect(itemElement.attributes('data-test-files')).toBe('2')
```

Comment:
any motive in using differently?

Reply:
just wandering how attributes can be accessed.
fixed

Reply:
then no need to test that double, this would only test the test-framework, no need to do that

Reply:
🤔, different keys are used.